packages/react-native-usb/src/ReactNativeUsb.types.ts

```diff
@@ -0,0 +1,82 @@
+// TODO: better types than any
```

Comment:
Do you plan to add types before merging?

Reply:
Probably not, we don't use these properties anywhere but I want them to be here if someone else would want to add/use them in future. This library could be used even with 3rd parties for different devices than Trezors.